db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/CompilerContext.java

```diff
@@ -607,6 +608,8 @@
 	 */
 	void addRequiredSchemaPriv(String schema, String aid, int privType);
 
+	void addRequiredAccessSchemaPriv(UUID uuid);
```

Comment:
Why does this new method require `UUID` while the one above `String`?

Reply:
Hi @jaceklaskowski Thanks for the comments. We could use addRequiredSchemaPriv(). A schema descriptor contains both UUID and schemaName, and they have 1-to-1 mapping. Passing down UUID is more of a coding convenience as StatementSchemaPermission uses UUID to do the authorization check. If passing down schemaname, we still need to set UUID for the StatementSchemaPermssion object created.